python-stubs/dolphin/_modules/memory.pyi

```diff
@@ -92,8 +80,7 @@ def read_f64(addr: int, /) -> float:
     :return: value as floating point number
     """
 
```

Comment:
When I added memory.invalidate_icache(u32 addr, u32 size), I forgot to add anything here. A rough description would be "Invalidates JIT cached code between the address and address + size, forcing to the JIT refetch instructions instead of executing from its cache."

Reply:
Does this look good to you?

```suggestion

def invalidate_icache(addr: int, size: int, /) -> None:
     """
     Invalidates JIT cached code between the address and address + size, forcing to the JIT refetch instructions instead of executing from its cache.
   
    :param addr: memory address to start invalidation at
    :param size: size of the cache as integer
    """

```

Reply:
Aside from accidentally having my grammar mistake "to the JIT" which should be "the JIT to", looks good